src/main.rs

```diff
@@ -66,28 +80,29 @@ impl<W: std::io::Write> RockPaperScissorsGame<W>{
         Ok(true)
       },
       shape_played => {
-        let result = self.evaluate_as_left(shape_played, self.computer_shape);
-        dbg!(
-          // "you played {}, they played {}, result = {}",
+        let result = self.evaluate_game_as_left(shape_played, self.computer_shape);
+        writeln!(
```

Comment:
Rad by som ta upozornil, ze robis viacero zmien v ramci jedneho commitu. Menis sposob vypisovania aj datovy model. Na to maju byt dva separatne commity. Teraz sa to blbo cita, lebo musim sledovat dve zmeny a pri kazdej sa zamysliet nad tym, ze ku ktorej zo semantickych zmien to patri.